web/src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import { VFC } from "react";
+
+import { Theme, Snackbar, Alert } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+interface Props {
+  show: boolean;
+  onClose: () => void;
+}
+
+export const ErrorNotification: VFC<Props> = ({ show, onClose }) => {
+  const theme = useTheme<Theme>();
+
+  return (
+    <Snackbar
+      anchorOrigin={{ vertical: "bottom", horizontal: "center" }}
+      open={show}
+      onClose={onClose}
+      sx={{ width: "764px", "& .MuiPaper-root": { py: 0.75 } }}
+    >
+      <Alert
+        onClose={onClose}
+        severity="error"
+        sx={{
+          width: "100%",
+          backgroundColor: theme.palette.error.main,
+          color: "common.white",
+        }}
+        icon={false}
+      >
+        Something went wrong. Please try again.
```

Comment:
text inside Alert shouldn't be hardcoded + it's looks like onClose calls 2 time (first in Snackbar props and second in Alert props)

Reply:
The thing is that <Snackbar /> as the most external and main component of this popup needs it for example to close it when we click outside this element. And <Alert /> as different component I putted inside gives me ‘X’ icon only when I pass onClose prop which is also saying what it should do onClose.